### PR TITLE
new hw entrance-box component

### PR DESCRIPTION
**Pull request by Charan42**
new entrance box component for frontpage.

### CODE REVIEW

**Comment by Charan42** [comment]
[new hedwig component for front page boxes with images](https://trello.com/c/cmH8oa0Z/266-new-hedwig-component-for-front-page-boxes-with-images)

File: src/shared/components/entrance-box/entrance-box.css [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    height: 100%;
+    background-position: center;
+    background-size: cover;
+    transition: all 0.3s cubic-bezier(0, 0, 0.1, 1);
+  }
+
+  &__body {
+    z-index: 1;
+    padding: var(--hw-spacing);
+    background: var(--hw-color-gray-lightest);
```
Comment (viggo) [inline src/shared/components/entrance-box/entrance-box.css]: I think we should use the new responsive background color here.

File: src/shared/components/entrance-box/entrance-box.css [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    &::after {
+      opacity: 1;
+    }
+  }
+
+  &:hover .hw-entrance-box__title {
+    border-bottom: 2px solid var(--hw-color-link-underline);
+    color: var(--hw-color-gray-darker);
+  }
+  &:hover .hw-entrance-box__text {
```
[the remark this answers is not in the document]
Comment (viggo) [reply src/shared/components/entrance-box/entrance-box.css]: It only transitions when starting to hover. When not hovering anymore it instantly gets no border. The transition should be directly on the __text and __label classes.
Also, the transition of the border is weird looking; going from dark gray to green and from thin to 2px.
__label and __text should have a default 2px solid white border-bottom.

[the remark this answers is not in the document]
Comment (Charan42) [reply src/shared/components/entrance-box/entrance-box.css]: having a default white border will be an issue when the text is placed on a lighter grey background. instead, I'll add a transparent border color

**Pull request merged**